### PR TITLE
adding swedish_medical_ner

### PR DESCRIPTION
**Pull request by bwang482**
Adding the Swedish Medical NER dataset, listed in "Biomedical Datasets - BigScience Workshop 2021"

Code refactored

### CODE REVIEW

**Review by albertvillanova — CHANGES_REQUESTED** [review]
The non passing test is about the dataset card. Some information is missing: 
```
        if error_messages:
>           raise ValueError("\n".join(error_messages))
E           ValueError: The following issues have been found in the dataset cards:
E           README Validation:
E           The following issues were found for the README at `/home/circleci/datasets/datasets/swedish_medical_ner/README.md`:
E           -	Expected some content in section `Data Fields` but it is empty.
E           The following issues have been found in the dataset cards:
E           YAML tags:
E           __init__() missing 1 required positional argument: 'pretty_name'
```

**Comment by bwang482** [comment]
Hi, what's the current status of this request? It says Changes requested, but I can't see what changes?

**Comment by lhoestq** [comment]
Hi, it looks like this PR includes changes to other files that `swedish_medical_ner`.

Feel free to remove these changes, or simply create a new PR that only contains the addition of the dataset

File: datasets/swedish_medical_ner/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Medical NER.
+
+### Languages
+
+Swedish (SV).
+
+## Dataset Structure
+
+### Data Instances
```
Comment (lhoestq) [inline datasets/swedish_medical_ner/README.md:73]: In this section we expect to see an actual example from the dataset, as it is when people use the dataset

Feel free to get one example using `load_dataset` and `dataset["train"][0]` and put it here :)

**Review by lhoestq — COMMENTED** [review]
Thanks a lot for adding this dataset ! The dataset card and dataset script look all good to me, good job :)

I just added one comment.

Also feel free to ping me when your have removed the other changes or created a new PR

File: datasets/swedish_medical_ner/README.md (hunk 1 above)
Comment (bwang482) [reply datasets/swedish_medical_ner/README.md:73]: Thanks @lhoestq ! I have created a new PR https://github.com/huggingface/datasets/pull/2940

**Pull request closed without merging**